docs/contributions.rst

```diff
@@ -84,72 +88,75 @@ e.g. ::
 C language
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Don't forget putting `static` modifiers. Don't introduce unnecessary
+Don't forget to use `static` modifiers. Don't introduce unnecessary
 global variables.
 
 Remove unused variables and types. If you want to keep them in your
-source code, put enough description as comments.
+source code, include a descriptive comment.
 
-Use available facility that ctags main part provides.  If the
-facility is not enough for writing a parser, consider extending the
-main part first.
+Use the available facilities provided by the ctags core. If the
+facilities are not enough for writing a parser, consider extending
+the core first.
 
 Use underscores in names only in file scope objects.
-Don't use it in function declarations, variable declarations
-and macro names in header files.
+Don't use them in function declarations, variable declarations or
+macro names in header files.
+
+Trim unnecessary white space at the end of a line. In emacs, `M-x
+whitespace-cleanup`.
 
-Trim unnecessary white space at the end of line. In emacs `M-x
-whitespace-cleanup`. Don't mix `whitespace cleanup` fix and the other
-improvements into one commit when changing the existing code. Style,
-including `whitespace cleanup` should be in a separate commit.
-Mixing makes reviewing harder.
+Don't mix `whitespace cleanup` fixes and other improvements in one
+commit when changing the existing code. Style fixes, including
+`whitespace cleanup`, should be in a separate commit. Mixing
+functional changes with style fixes makes reviewing harder.
```

Comment:
BTW, @vhda, @dkearns, how do you think this rule?
As I wrote before, this rule helps reviewers.